doc/content/source/materials/ConcreteThermalMoisture.md

```diff
@@ -0,0 +1,18 @@
+<!-- MOOSE Documentation Stub: Remove this when content is added. -->
```

Comment:
Ordinarily, I wouldn't want to merge new code in with a stub like this, but since you have the other PR up that adds this documentation, it's OK.  You'll need to revise the other PR to modify this file rather than the old one.